### PR TITLE
Consume DevOps Logging

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Consume DevOps Logging from eng\common\scripts\logging.ps1

### CODE REVIEW

File: eng/scripts/Verify-And-Merge-PRs.ps1 [hunk 1]
```diff
@@ -5,25 +5,7 @@ param(
   [switch]$devOpsLogging = $false
```
Comment (weshaggard) [inline eng/scripts/Verify-And-Merge-PRs.ps1:5]: I think we can eliminate this parameter as well now.

**Review by weshaggard — APPROVED** [review]
One comment but looks good otherwise.

**Review by mitchdenny — APPROVED** [review]

File: eng/scripts/Verify-And-Merge-PRs.ps1 [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-function LogError
-{
-  if ($devOpsLogging) {
-    Write-Host "##vso[task.logissue type=error]$args"
-  }
-  else {
-    Write-Error "$args"
-  }
-}
+. "${PSScriptRoot}..\common\scripts\logging.ps1"
```
Comment (mitchdenny) [inline eng/scripts/Verify-And-Merge-PRs.ps1:7]: nit: PowerShell conventions would make this ```Logging.ps1```

**Review by mitchdenny — APPROVED** [review]

File: eng/scripts/Verify-And-Merge-PRs.ps1 (hunk 2 above)
Comment (chidozieononiwu) [reply eng/scripts/Verify-And-Merge-PRs.ps1:7]: Renaming in git where the only change is a case change ends up causing more troubles than benefit. I'll just leave this.

Comment (weshaggard) [reply eng/scripts/Verify-And-Merge-PRs.ps1:7]: Yes we can potentially clean some of this up with https://github.com/Azure/azure-sdk-tools/issues/829.

**Pull request merged**